crates/interpreter/src/exec.rs

```diff
@@ -209,7 +211,8 @@ impl<'m> Store<'m> {
     /// may be corrupted.
     pub fn invoke<'a>(
         &'a mut self, inst: InstId, name: &str, args: Vec<Val>,
-    ) -> Result<RunResult<'a, 'm>, Error> {
+    ) -> Result<RunResult<'a, 'm>, Error>
+    where 'a: 'm {
```

Comment:
Same here. We should never have `'a: 'm`. At best we can have `'m: 'a` but it should be implicit. We shouldn't need to write it.